frontend/src/modules/teamcomments/components/Count.js

```diff
@@ -18,8 +18,32 @@ export default function Count(props: Props) {
     }
 
     const commentCount = teamComments.comments.length;
+    const pinnedCommentCount = teamComments.comments.filter(comment => {
+        return comment.pinned === true;
+    }).length;
+
+    if (!commentCount && !pinnedCommentCount) {
+        return null;
+    }
+
+    const pinned = (
+        !pinnedCommentCount ? null :
+        <span className='pinned'>{ pinnedCommentCount }</span>
+    )
+
+    const commentTotal = (
+        !commentCount ? null :
+        <span>{ commentCount }</span>
+    )
+
+    const dash = (
```

Comment:
I wasn't sure what character would be best as a separator here. Went with the dash but let me know if you prefer something different.

Reply:
Just to clarify ~ I currently have the counts as `<# of pinned comments>-<total comments>`.  
Should that be `<# of pinned comments>+<# of remaining comments>` instead?

I didn't originally set it up this way since we are only adding the pin icon and not sorting the pinned comments to the top so it felt more like "there are pinned comments within these comments" and less like "here are the pinned comments and here is everything else".

Reply:
>  Should that be `<# of pinned comments>+<# of remaining comments>` instead?

Yes. That would be consistent with the Machinery and Locales tabs and allow for consistent styling.